tests/test_converter.py

```diff
@@ -209,3 +212,43 @@ def test_example_4():
     })
 
     assert qubo == expected
+
+
+def test_to_dqm():
+    num_variables = 2
+    variables = sympy.symbols(
+        " ".join([f"x{i}" for i in range(num_variables)])
+    )
+    objective_function = from_sympy(variables[0] + variables[1])
+    print(objective_function)
+
+    constraint_le = Constraint(objective_function, [Polynomial(1)],
+                               Operator.LE, MethodsForInequalities.SLACKS_LOG_2, "s",)
+
+    problem = SimpleProblem(
+        objective_function, constraint_le,
+        MethodsForInequalities.SLACKS_LOG_2)
+
+
+    dqm = Converter.to_dqm(problem)
+
+    assert isinstance(dqm, DiscreteQuadraticModel)
```

Comment:
such test won't catch much :/ check if values are correct. Same below

Reply:
It's hard to get some values from the DQM instance (at least for me hehe) that could be compared without having variable objects, can I use SimpleProblem() instead of CommunityDetectionProblem(), or is it better to stay with SimpleProblem()? It would definitely make it easier in some way

Reply:
you can create DQM instance by hand and compare it (or at least i think that you should be able to compare that)

Reply:
In both cases, DQM and CQM, even when I create a comparison using the converter method `to_dqm()` and `to_cqm()` with exactly the same data (e.g. `dqm == handmade_dqm`), it returns `False`. The only thing that can be compared and will return True is the `variables` variable. 
What else can be done?

Reply:
That's sound good enough :p Just remove the first assert.
```suggestion
```